e2e-tests/users/run

```diff
@@ -6,6 +6,34 @@ set -o xtrace
 test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 
+is_password_updated() {
+	local username=$1
+	local uri=$2
+	run_mysql "SELECT User_attributes FROM mysql.user WHERE user='${username}'" "${uri}" | grep additional_password
+}
+
+is_old_password_discarded() {
+	local username=$1
+	local uri=$2
+	run_mysql "SELECT User_attributes FROM mysql.user WHERE user='${username}'" "${uri}" | grep NULL
+}
+
+wait_for_password_propagation() {
+	local secret=$1
+	local user=$2
+	local root_pass=$(getSecretData "${secret}" "root")
+
+	until is_password_updated "${user}" "-h ${cluster}-pxc -uroot -p'${root_pass}'"; do
```

Comment:
@egegunes how many times do we need to wait? we can wait hours ... but without any results

Reply:
it was done